src/components/anchor-navigation/anchor-navigation.stories.mdx

```diff
@@ -0,0 +1,98 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+
+import { AnchorNavigation, AnchorNavigationItem, AnchorSectionDivider } from '.';
+import Textbox from '../../__experimental__/components/textbox';
+
+<Meta
+  title="Design System/Anchor Navigation"
+  parameters={{ info: { disable: true }}}
+/>
+
+# Anchor Navigation
+Easily navigate through long scrollable content.
+
+## Overview
+
```

Comment:
If there is no content in these sections you can remove the headings.

Reply:
I actually left it there because I thought it would be good to add it in the future - so I left it as a reminder of work needed to be done. I can remove though if you think that's better

Reply:
I'd say either add the content now or remove the headings.